### PR TITLE
Positions page rows

### PR DESCRIPTION
**Pull request by phoebemirman**
- add market row and expandable row components
- add numbers updating on filter box

### CODE REVIEW

**Comment by AugurIntegration** [comment]
This pull request **introduces 7 alerts** and **fixes 1** when merging 09a142940377b21ac1b703eeb8980590fed5dbaa into faf062966c3b173c792613e8baaa39b2f1fe75bc - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-288d6bd550c82b46d86fdb2019930cf78bae7542)

**new alerts:**

* 7 for Unused variable, import, function or class

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/modules/portfolio/components/common/market-row.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    }
+    rowContent={
+      <div className={Styles.MarketRow__infoContainer}>
+        <span className={Styles.MarketRow__description}>{props.market.description}</span>
+        <span className={Styles.MarketRow__time}>{props.market.creationTime.formattedShortDate}</span>
+      </div>
+    }
+    toggleContent={
+      <div>
+        info
```
Comment (bthaile) [inline src/modules/portfolio/components/common/market-row.tsx]: what is `info` here?

File: src/modules/portfolio/components/markets/markets.jsx [hunk 2]
```diff
@@ -28,26 +29,27 @@ function filterComp(input, market) {
...
-    location: PropTypes.object.isRequired,
+    //   loadMarketsInfo: PropTypes.func.isRequired,
+    //   location: PropTypes.object.isRequired,
     myMarkets: PropTypes.object.isRequired,
-    toggleFavorite: PropTypes.func.isRequired,
-    pendingLiquidityOrders: PropTypes.object.isRequired,
-    outcomes: PropTypes.object.isRequired,
+    //   toggleFavorite: PropTypes.func.isRequired,
+    //   pendingLiquidityOrders: PropTypes.object.isRequired,
+    //   outcomes: PropTypes.object.isRequired,
```
Comment (bthaile) [inline src/modules/portfolio/components/markets/markets.jsx]: can we get rid of the commented out stuff?

File: src/modules/portfolio/components/common/market-row.tsx (hunk 1 above)
Comment (phoebemirman) [reply src/modules/portfolio/components/common/market-row.tsx]: that is just showing where the expanded content will go

**Pull request merged**